src/v3/package.json

```diff
@@ -39,15 +39,16 @@
     "ie 11"
   ],
   "dependencies": {
+    "@adobe/leonardo-contrast-colors": "1.0.0-alpha.13",
     "@babel/runtime": "^7.18.6",
     "@emotion/cache": "^11.9.3",
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
     "@hcaptcha/react-hcaptcha": "^1.8.1",
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.8.5",
-    "@okta/odyssey-design-tokens": "^1.2.0",
-    "@okta/odyssey-react-mui": "^1.2.0",
+    "@okta/odyssey-design-tokens": "1.9.1",
+    "@okta/odyssey-react-mui": "1.9.1",
```

Comment:
You're intentionally pinning to the exact semver?

Reply:
Yeah I decided to pin exactly because even though Odyssey promised no breaking changes they have still been having some breaking changes. For example, the `OdysseyProvider` API was broken for a few versions and only just got fixed at this `1.9.1` update